tests/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+document: test-metadata
+subtype: general
+enable_lint: True
```

Comment:
This means, like dockerfilelint.py, which we have already in MTF?

Reply:
yep, it enbles all linters, whaever it means in sense of running MTF
In details it runs all linters, including "dockerfilelint.py"